src/styles/_catalog.scss

```
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+    transition: transform 300ms, color 300ms;
```

Create a variable for this value, you use it in the several places
```suggestion
    transition: transform $transition-duration, color $transition-duration;
```